### PR TITLE
pipewire: 0.3.33 -> 0.3.34

### PR DESCRIPTION
**Pull request by Kranzes**
###### Motivation for this change
1. Updated package to the latest version
2. Imported the latest json configs into the module
3. Ran nixpkgs-fmt on the expression
###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/#sec-conf-file))
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by jansol — APPROVED** [review]

File: pkgs/development/libraries/pipewire/default.nix [hunk 1]
```diff
@@ -27,28 +27,37 @@
 , callPackage
 , nixosTests
 , withMediaSession ? true
-, gstreamerSupport ? true, gst_all_1 ? null
-, ffmpegSupport ? true, ffmpeg ? null
-, bluezSupport ? true, bluez ? null, sbc ? null, libfreeaptx ? null, ldacbt ? null, fdk_aac ? null
+, gstreamerSupport ? true
+, gst_all_1 ? null
```
Comment (oxalica) [inline pkgs/development/libraries/pipewire/default.nix:31]: Do we really need `? null` for all optional dependencies? 
They are available by `callPackage` and are lazy evaluated.

Comment (jansol) [reply pkgs/development/libraries/pipewire/default.nix:31]: I'm fairly sure that is just a "that's just how it has always been done" case. At least for me it has, it was that way for existing optional deps so I used the same pattern for the ones I added.

**Comment by archseer** [comment]
Can we get this merged? It resolves a few critical regressions on 0.3.33.

**Comment by flexagoon** [comment]
@jtojnar  could you please review this?

**Pull request merged**